### PR TITLE
add output message when nupkg is packed successfully

### PR DESCRIPTION
**Pull request by rohit21agrawal**
Fixes: https://github.com/NuGet/Home/issues/4294

CC: @emgarten @rrelyea @alpaix @mishra14 @zhili1208 @jainaashish @nkolev92

### CODE REVIEW

File: src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets [hunk 1]
```diff
@@ -124,7 +127,9 @@ Copyright (c) .NET Foundation. All rights reserved.
               ContentTargetFolders="$(ContentTargetFolders)"
               RestoreOutputPath="$(RestoreOutputAbsolutePath)"
               NuspecFile="$(NuspecFileAbsolutePath)"/>
+    <Message Text="Project packed successfully. Output available at $(PackageOutputAbsolutePath)" Importance="High"/>
```
Comment (emgarten) [inline src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets]: Should this be localized?

File: src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets [hunk 2]
```diff
@@ -81,6 +81,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ConvertToAbsolutePath Paths="$(RestoreOutputPath)">
       <Output TaskParameter="AbsolutePaths" PropertyName="RestoreOutputAbsolutePath" />
     </ConvertToAbsolutePath>
+    <ConvertToAbsolutePath Paths="$(PackageOutputPath)">
+      <Output TaskParameter="AbsolutePaths" PropertyName="PackageOutputAbsolutePath" />
```
Comment (emgarten) [inline src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets:85]: should ``PackageOutputPath="$(PackageOutputPath)"`` be changed also?

**Review by emgarten — CHANGES_REQUESTED** [review]
questions on localization

File: src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets (hunk 2 above)
Comment (rohit21agrawal) [reply src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets:85]: fixed

**Comment by rohit21agrawal** [comment]
@emgarten i used the underlying msbuild logger to workaround the localization issue, thanks for your suggestion :)

**Review by emgarten — APPROVED** [review]

**Pull request merged**